### DISCUSSION TITLE
How to know the installed version of pivpn scripts?

### DISCUSSION THREAD

**Discussion by TrustCane** [pivpn/pivpn#1781] [Q&A] [answered]
Hi all. How can I find out what version of pivpn I have installed? I had WG installed on different VPS at different times via PiVPN. I did "apt update; apt upgrade" on both servers, but the older one still has the "[start of names](https://github.com/pivpn/pivpn/commit/f9ae9662ecc49896d09cdb38e177d2efa90e7119)" matching bug, while the new one does not. I don't understand why, since I updated both servers. Would like to check the pivpn versions.

**Comment by orazioedoardo** [comment]
Check the commit in the `pivpn debug` output.

**Reply by TrustCane** [reply]
The commit on my first server says "Date: Fri Dec 3 21:41:13 **2021** +0100".
The commit on my last server (I installed WG pivpn later than everyone else) says "Date: Thu Aug 3 23:33:23 2023 +0200".

I don't understand why the commit on the first server is from 2021, if I regularly perform `apt update; apt upgrade.` So the scripts are not updated?

**Reply by coolapso** [reply, marked as answer]
Because pivpn scripts are not updated via package manager. and there is no real need for that. 

What is important to keep up to date is the VPN protocol (wireguard / openvpn) and those you update via APT  ... .

https://github.com/pivpn/pivpn/wiki/Upgrading#how-do-we-upgrade-pivpn
